packages/larva/scripts/build-icons.js

```diff
@@ -5,15 +5,21 @@ const path = require('path');
 const mkdirp = require('mkdirp');
 const globby = require('globby');
 const fs = require('fs');
```

Comment:
@laras126 Is this ready to merge?
I can see many comments.

Reply:
Yes - those are for a partially developed feature that we might come back to.